src/client/app/containers/app.ts

```diff
@@ -73,12 +73,13 @@ export class AppComponent {
         return this.profile$
             .filter(profile => profile != null)
             .map(profile => {
-                if (!isNil(this.snippet.gistOwnerId)) {
-                    return this.snippet.gistOwnerId === profile.login;
+                if (isNil(this.snippet.gist)) {
```

Comment:
Curious. Can you link me to the implement of `isNil`. Would `_.isEmpty` not work?